frontend/javascripts/admin/user/experience_modal_view.js

```diff
@@ -1,6 +1,7 @@
 // @flow
 
-import { Modal, Button, Tooltip, Icon, Table, InputNumber, Tag, Badge } from "antd";
+import { Modal, Button, Tooltip, Table, InputNumber, Tag, Badge } from "antd";
+import { CloseOutlined, DeleteOutlined, RollbackOutlined } from "@ant-design/icons";
 import * as React from "react";
 import _ from "lodash";
 
```

Comment:
This Modal does not look well. The experience entry with domain name, value and bin icon is not aligned.